content/en/blog/2020-10-07-helm-hub-moving-artifact-hub.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Helm Hub Moving To Artifact Hub"
+slug: "helm-hub-moving-artifact-hub"
```

Comment:
Do you think adding "to" to the slug might help disambiguate it? I worry that "those Helm Hub people are moving Artifact Hub to somewhere" is an incorrect reading that the missing "to" might encourage.

Reply:
done